### PR TITLE
Add integration test for place order and time-locked token contract

### PR DESCRIPTION
**Pull request by yzang2019**
## Describe your changes and provide context
**Goal:**
Use github workflow + docker for CI integration test

**Change:**
As the first step, we add a simple test for testing dex module including:
1. Register a contract
2. Place an order
3. Verify order is placed successfully

We also added the verification and integration test for timelocked token contract and split into 3 test files.

## Testing performed to validate your change
Run and tested locally and verified it works

### CODE REVIEW

**Review by philipsu522 — APPROVED** [review]

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -38,4 +40,16 @@ jobs:
...
       - name: Verify Sei Chain is able to start up
-        run: ./docker/integration_test/startup_test.sh
+        run: docker exec -i sei-node-0 integration_test/startup/startup_test.sh
+
+      - name: Testing Dex Module
+        run: docker exec -i sei-node-0 integration_test/dex_module/place_order_test.sh
+
+      - name: Testing Wasm Module
+        run: |
+          docker exec -i sei-node-0 integration_test/contracts/deploy_timelocked_token_contract.sh
```
Comment (yzang2019) [inline .github/workflows/integration-test.yml:50]: We will initiate new contracts before each test scenario

**Review by Eko101089 — COMMENTED** [review]
Why this ?

**Review by codchen — APPROVED** [review]
are the mars etc. wasm builds intentionally included in the commit?

**Comment by yzang2019** [comment]
> are the mars etc. wasm builds intentionally included in the commit?

The PR was initially for dex module testing, so mars were added for that. Then I repurpose it to also include wasm module

**Comment by yzang2019** [comment]
> Why this ?

Why what?

**Review by philipsu522 — COMMENTED** [review]
- Let's change https://github.com/sei-protocol/sei-chain/pull/836/files#diff-396c970b862a5f8668dd8016364b8c5b58b942dc47846ce01e250e5df7954908L19 to be 2 to be consistent
- do we want to check in the wasm files? or have them in s3 or something

**Comment by yzang2019** [comment]
> * Let's change https://github.com/sei-protocol/sei-chain/pull/836/files#diff-396c970b862a5f8668dd8016364b8c5b58b942dc47846ce01e250e5df7954908L19 to be 2 to be consistent
> * do we want to check in the wasm files? or have them in s3 or something

It's already on s3, but put it here for convenience, is there any risk adding these files?

**Pull request merged**